src/Client.php

```diff
@@ -42,8 +40,60 @@ public function createClient($apiKey, $useBundle, $endpoint): ApiClient
             $apiKey,
             $useBundle ?
                 [
-                    CURLOPT_CAINFO => self::getCaCertPath()
+                    CURLOPT_CAINFO => self::CA_CERT_FILE_PATH
                 ] : []
         );
     }
+
+    public function checkAvailabilityForPaymentMethodUsingCurrency(string $payment_method_name, Currency $currency): bool
+    {
```

Comment:
Just a question. Will be added the ability to clear cached multicurrency?

Reply:
Good question, i think i can use it to make a method, which removes the cached file. We will be able to use this method as trigger to start update for caching multi currency which planing to use after save API key actions.